### PR TITLE
Update Do not caps-protect name fields #384.biblatex

### PR DESCRIPTION
**Pull request by nickbart1980**

### CODE REVIEW

File: test/fixtures/export/Do not caps-protect name fields #384.biblatex [hunk 1]
```diff
@@ -27,11 +28,13 @@
...
 
 @book{bovendeert,
-  author = {{boven d'Eert}, Christianus}
+  author = {boven d' Eert, Christianus},
+  options = {useprefix=true}
 }
 
 @book{s-gravesande,
-  author = {'s-Gravesande, Goverdus}
+  author = {'s- Gravesande, Goverdus},
```
Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.biblatex]: Should a dash always be followed by a space in BibLaTeX?

Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.biblatex]: If at the end of a prefix/particle: Yes.

Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.biblatex]: Or should a particle simply *always* be followed by a space?

Comment (nickbart1980) [inline test/fixtures/export/Do not caps-protect name fields #384.biblatex]: Yes, indeed.

Comment (retorquere) [inline test/fixtures/export/Do not caps-protect name fields #384.biblatex]: OK, tests are running on that at https://travis-ci.org/ZotPlus/zotero-better-bibtex/jobs/88983180

**Pull request merged**